.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+on:
+  push:
+    tags:
+      - "v0.*"
+jobs:
+  publish:
+    environment:
+      name: maven-central-and-ruby-gems
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up OpenJDK 8
+      uses: actions/setup-java@v4
+      with:
+        java-version: 8
+        distribution: "temurin"
+    - name: Publish
+      run: |
+        mkdir -p $HOME/.gem
+        touch $HOME/.gem/credentials
+        chmod 0600 $HOME/.gem/credentials
+        printf -- "---\n:rubygems_api_key: ${RUBYGEMS_API_KEY}\n" > $HOME/.gem/credentials
+        ./gradlew --stacktrace publishMavenPublicationToMavenCentralRepository gemPush
+      env:
+        ORG_GRADLE_PROJECT_ossrhUsername: ${{ secrets.OSSRH_USERNAME }}
```

Comment:
We recently use `vars` instead of `secrets` for username. (See some other repos.)

Reply:
Thanks, I copied these parts.

https://github.com/embulk/embulk-decoder-bzip2/blob/3783be42c7cedc42c3c73c677f3e50451a4d9b8b/.github/workflows/publish.yml#L24-L27